src/Common/Misc/ConvertProxyProperty/index.js

```diff
@@ -9,7 +9,7 @@ const typeMapping = {
   ProxyEditorPropertyWidget: 'ProxyEditorPropertyWidget',
 };
 
-function isGroupWidget(widgetType) {
+export function isGroupWidget(widgetType) {
```

Comment:
I'm not sure you need to export that function since it is part of the default export.